src/sidebar-overlay/components/Annotation.css

```diff
@@ -240,6 +240,52 @@ input:focus {
     outline: none;
 }
 
+.shareIcon {
+    box-sizing: border-box;
+    background-image: url('/img/share.svg');
+    background-repeat: no-repeat;
```

Comment:
Just a quick note: all these icon classes repeat a similar set of rules. Rather than repeating them in each class, create a common class with the shared rules which you can apply to all of the icon elements